src/App.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import './App.css';
+import resumeJson from './assets/resume.json';
+
+import Experience from "./components/Experience/Experience";
+import Study from "./components/Study/Study";
+import Award from "./components/Award/Award";
+import Skill from "./components/Skill/Skill";
+import Language from "./components/Language/Language";
+import Interest from "./components/Interest/Interest";
+import Reference from "./components/Reference/Reference";
+
+function App() {
+  const resumeData = resumeJson;
+  return (
+    <main className="App">
+      <header>
+        <div>
+          <h1>{resumeData.basics.name}</h1>
+          <span className="label">{resumeData.basics.label}</span>
```

Comment:
It would be good to move the basics information into a separate component

Reply:
I'm still a newb and thats also what I did, but I would like to see what a professional has to say.